### ISSUE TITLE
suggestions on new functions of OSS

### ISSUE THREAD

**Issue by coOloUtbReaKer** [GTNewHorizons/GT-New-Horizons-Modpack#12736]
### Your GTNH Discord Username

coOloUtbReaKer

### Your Pack Version

2.2.10

### Your Proposal

Object Submission Stations of BetterQuesting needs some new features because it is difficult to use.
1.Add a research button for OSS.
2. automatic IO is needed.

### Your Goal

first, we can't search a specific quest and instead we can only find quest by click button for countless times because of GTNH's thonsands of quests. that's a terrible experience. 
second, it's very funny to create OSS, find specific quests, complete the quest, and finally, open the quest book to receive the rewards. things get complicated on the contrary via OSS. If we can select a specific quest, use item pipes to put in the required items, and output the rewards automatically, OSS can be then be very useful. even if it is put in LV or even higher tier.

### Your Vision

OSS is not very useful. waiting for CD of repeatable quests is a waste of time, and players can make full use of coins before tons of coins become completely useless and be thrown into trash bin.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by chochem** [comment]
the other option would be to make sure its not needed anywhere and then disable it. (I think LESU might be the only one right now?)

**Comment by King65868** [comment]
Not even needed anymore since those parts are now checkmark tasks

**Comment by chochem** [comment]
right. I forgot that I changed that already. So I guess we could disable it then.

**Comment by miaowwwwww** [comment]
I think this suggestion is very useful as coin are only useful in the early game and to use them efficiently u have to manually submit every time, it is painful.
If using coins, early game can be boosted greatly, and i think this suggestion does just that.

a little more info on the OSS right now:
It can be used to submit items, but it does not complete quests for the player, players have to click submit themselves and get reward.
After a quest is completed, the OSS can't find the quest anymore, so players have to wait till the cool down is over and select that quest again.

So, changes needed:
1.search
2.make quests lockable
3.automatic complete, and auto IO.

**Comment by chochem** [comment]
not relevant for coins anymore
